Tests/Unit/MSFT_xFirewall.Tests.ps1

```diff
@@ -149,6 +149,7 @@ try
                 }
             }
             Context 'Ensure is Present and the Firewall is Absent' {
+                Mock Get-FirewallRule
```

Comment:
One more quick word about this:
As a test I thought I'd let this run through every rule on my Win 10 desktop (with your above fix) thinking that there would be no failures. Unfortunately there were 2 failed tests out of 45693 tests:
![2015-11-18_16-00-58](https://cloud.githubusercontent.com/assets/7589164/11231498/ea80f350-8e0d-11e5-8205-5f97dedeed36.png)

I didn't output the nunit XML here though so I can't find the failed tests until I get a chance to rerun them (which takes about 4 hours). I'll do this tomorrow and track down what ever bug still remains. I'll log this in a separate issue/PR when I find it.